### DISCUSSION TITLE
第5节课笔记(11班)

### DISCUSSION THREAD

**Discussion by keesh0410** [InternLM/Tutorial#304] [笔记]
要记得群里的打分表也要填哟！

**Comment by seifer08ms** [comment]
https://zhuanlan.zhihu.com/p/677337189

**Comment by zhanghui-china** [comment]
[https://zhuanlan.zhihu.com/p/677583802](url)

**Comment by qianlian-mozi** [comment]
https://blog.csdn.net/m0_54824511/article/details/135573678

**Comment by lvoooo** [comment]
https://github.com/lvoooo/internLM-learning/blob/main/5.md
https://github.com/lvoooo/internLM-learning/blob/main/5.1.md

**Comment by horacehht** [comment]
https://blog.csdn.net/m0_50123831/article/details/135571393

**Comment by BarryYin** [comment]
https://github.com/BarryYin/internLM/blob/main/%E7%AC%AC%E4%BA%94%E8%8A%82.md

**Comment by Shengshenlan** [comment]
https://github.com/Shengshenlan/tutorial/blob/master/note/5/%E7%AC%AC5%E8%8A%82%E8%AF%BE%E7%AC%94%E8%AE%B0.md

**Comment by wwewwt** [comment]
https://github.com/wwewwt/-InternLM/wiki/5%E3%80%81LMDeploy-%E7%9A%84%E9%87%8F%E5%8C%96%E5%92%8C%E9%83%A8%E7%BD%B2%E8%AF%BE%E7%A8%8B-%E4%BD%9C%E4%B8%9A

**Comment by morning-start** [comment]
https://flowus.cn/morningstar/share/64d5a752-5d21-4e19-995a-b8096aa1a496
【FlowUs 息流】LMDeploy大模型量化部署实践

**Comment by wuyingxue1230** [comment]
https://zhuanlan.zhihu.com/p/677972589?

**Comment by JustLike1** [comment]
https://blog.csdn.net/qq_45607888/article/details/135557459
